テーブル設計

wataru-ando opened the pull request:
# WHAT

DB設計
# WHY

あとあと変更するのは手間がかかるから

Commit:
テーブル設計

wataru-ando committed:
update,README.md

改行などができていなかったため

wataru-ando committed:
update,README.md

改行などができていなかったため

hc0208 commented on README.md:
```diff
@@ -0,0 +1,65 @@
+~~~~~~~~~~~~~~~  
+proto_image_url :text  
+product_id : integer  
+~~~~~~~~~~~~~~~  
```
likesテーブルも欲しいかな

hc0208 commented on README.md:
```diff
@@ -0,0 +1,65 @@
+1,productsテーブル  
+アソシエーション
+~~~~~~~~~~~~~~~  
+belongs_to :uses  
```
userかな

hc0208 commented on README.md:
```diff
@@ -0,0 +1,65 @@
+~~~~~~~~~~~~~~~  
+belongs_to :uses  
+has_many :comment  
+has_many :image  
```
commentもimageも複数形にしてほしい

hc0208 commented on README.md:
```diff
@@ -0,0 +1,65 @@
+update_at : timestamp  
+user_id : integer  
+image_id : integer  
+~~~~~~~~~~~~~~~  
```
_idはreferences型にしてほしい

hc0208 commented on README.md:
```diff
@@ -0,0 +1,65 @@
+~~~~~~~~~~~~~~~  
+カラム  
+~~~~~~~~~~~~~~~  
+proto_image_url :text  
```
urlだけで良さそう

wataru-ando committed:
update,README.md

・likesテーブルの追加
・_idの方をintegerからreferencesに変えた
・has_manyで単数系だったところを複数形にした
・proto_image_urlと、カラム名が長かったからurlに簡略化した
